release/rancher/rancher.go

```diff
@@ -826,6 +839,24 @@ func registryAuth(authURL, service, image string) (string, error) {
 	return auth.Token, nil
 }
 
+func authFromEnv() (*tokenRequest, error) {
+	username := os.Getenv("DOCKER_USERNAME")
```

Comment:
Should these be part of the overarching config?

Reply:
Yes, I moved it to the cmd

Reply:
Updated to use flags